RELEASE_HISTORY.md

```diff
@@ -0,0 +1,84 @@
+- **Maintenance**: Following the Active phase, each major version remains in this phase for an additional 2.5 years, receiving only critical fixes and security patches.
+- **Under Development**: The pre-Active phase where the next major version is prepared. This phase varies in length and is not available for general consumption.
+
+### Guarantees
+
+- **Critical Defects Fixes**: The community will fix critical defects. The criteria for what constitutes a critical defect can be found [here](https://github.com/zowe/community/blob/master/Technical-Steering-Committee/release.md#active-release).
+- **Extender Conformance**: Extenders achieving Zowe conformance for the long-term support version will not need to modify their product to remain functional when Zowe updates are provided within the same major version.
+
+### Recommendations
+
+- **Production**: Use **Active** or **Maintenance** releases for production due to the guaranteed stability and the community’s commitment to fixing critical defects.
+- **Nightly Builds**: Available for integration testing. Use at your own risk.
+
+## Zowe Release Process
+
+### Short Summary
+
+1. Code Freeze date is hit.
+2. Each [squad](https://github.com/zowe/community/blob/master/Technical-Steering-Committee/squads.md) provides a version of the code to integrate into the Release Candidate (RC).
+3. RC is built and tested with an automatic test suite.
+4. RC is deployed and tested by squads.
+5. RC is approved by the TSC vote.
+6. Release is published.
+7. Documentation is published.
+8. Release retrospective is held.
+
+### Release Numbering
+
+Releases follow [semantic versioning](https://semver.org/) guidelines (MAJOR.MINOR.PATCH).
+
+- **MAJOR**: Incompatible API changes.
+- **MINOR**: Backwards-compatible functionality additions.
+- **PATCH**: Backwards-compatible bug fixes.
+
+### Release Content
+
+The following [components of the release](https://github.com/zowe/community/blob/master/Technical-Steering-Committee/release.md#release-content) are managed by the CLI squad:
+
+- **CLI Component**
+  - CLI Core
+  - CLI Plugins
+- **Client SDKs**
+  - Node.js Client SDK
+  - Python Client SDK
+
+
+## Zowe CLI Squad
+
+| Name | GitHub ID | Email | Roles |
+|-|-|-|-|
+| Adam Wolfe | @adam-wolfe | adam.wolfe@broadcom.com | Lead, TSC Rep, Security Squad Rep |
+| Amanda D'Errico | @AmandaDErrico | amanda.derrico@ibm.com | Developer |
+| Amber Torrise | @ATorrise | amber.torrise@broadcom.com | Developer |
+| Ana X. Ceron | @anaxceron | ana.ceron@broadcom.com | Tech Writer |
+| Andrew Harn | @awharn | andrew.harn@broadcom.com | Developer |
+| Eugene Johnston | @gejohnston | eugene.johnston@broadcom.com | Developer |
+| Fernando Rijo Cedeno | @zFernand0 | fernando.rijocedeno@broadcom.com | Developer |
+| Jessica Tonda | @JTonda | jessica.tonda@broadcom.com | Scrum Master |
+| Kevin Loesch | @KevinLoesch1 | kevin.loesch@broadcom.com | Developer |
```

Comment:
Should Kevin still be on this list? What about Jace?

Reply:
This information is sourced from https://github.com/zowe/community/blob/master/COMMITTERS.md#zowe-cli-squad, we can open a PR on the community repo to update the list of names 😋

Reply:
ill open a pr :)

Reply:
PR was merged. If the change could be reflected here, it would be appreciated. 🎊

Reply:
i realize there's actually an issue with the way this table is being created. Ana's name is breaking it because she has 'X.' for her middle name. I will fix that and make sure periods don't mess anything up and will update the table for this PR in my next commit
![image](https://github.com/user-attachments/assets/92c1cf09-381e-4abb-b451-ea81a3722256)